lib/Mandel/Model.pm

```diff
@@ -125,6 +125,8 @@ sub _add_field {
 
     # We compile custom attribute code for speed
     no strict 'refs';
+    # To fix warnings in coercion
+    no warnings;
```

Comment:
Not sure if I will accept either, but at least try to come up with something a bit more specific than turn off everything.